test/native-tests/test-char/Platform.js

```diff
@@ -33,15 +33,15 @@ ifeq (darwin, ${PLATFORM})\n\
     $(ICU4C_LIBRARY_PATH)/lib/libicuuc.a \
     $(ICU4C_LIBRARY_PATH)/lib/libicui18n.a\n\
 else\n\
-\tCFLAGS=-lstdc++ -std=c++0x -I$(IDIR)\n\
+\tCFLAGS=-lstdc++ -I$(IDIR)\n\
```

Comment:
This is just another one of the tests. We have other tests those we compile with `c++` flags. Since we test `C` files for this particular test, no `c++` flag is allowed / needed.